pkg/scheduler/framework/plugins/volumebinding/binder_test.go

```diff
@@ -540,8 +540,8 @@ func (env *testEnv) validateBind(
 		// Cache may be overridden by API object with higher version, compare but ignore resource version.
 		newCachedPV := cachedPV.DeepCopy()
 		newCachedPV.ResourceVersion = pv.ResourceVersion
-		if !reflect.DeepEqual(newCachedPV, pv) {
-			t.Errorf("cached PV check failed [A-expected, B-got]:\n%s", diff.ObjectDiff(pv, cachedPV))
+		if !reflect.DeepEqual(pv, newCachedPV) {
+			t.Errorf("cached PV check failed [A-expected, B-got]:\n%s", diff.ObjectDiff(pv, newCachedPV))
```

Comment:
Hm, I don't think this change is needed since `cachedPV` is the PV we get actually.